internal/cmd/billing/command_cost_list.go

```diff
@@ -82,19 +82,18 @@ func (c *command) list(cmd *cobra.Command, args []string) error {
 
 		// These fields may be empty depending on the line type, so casting floats as strings as to avoid zero-value
 		if price, ok := cost.GetPriceOk(); ok {
-			out.Price = utils.FormatPrice(*price, cost.GetUnit())
+			out.Price = billing.FormatPrice(*price, cost.GetUnit())
 		}
 
 		if originalAmount, ok := cost.GetOriginalAmountOk(); ok {
-
-			out.OriginalAmount = fmt.Sprintf("%.8f", *originalAmount)
+			out.OriginalAmount = *originalAmount
 		}
```

Comment:
Is `out.OriginalAmount = cost.GetOriginalAmount()` equivalent?

Reply:
ah yes i originally did this hoping that i could use pointer values to determine nil vs non-nil value but I dont think that's supported as i mentioned [here](https://github.com/confluentinc/cli/pull/1903#discussion_r1218794121)